docs/podman.1.md

```diff
@@ -117,15 +117,17 @@ Print the version
 
 **libpod.conf** (`/etc/containers/libpod.conf`)
 
-libpod.conf is the configuration file for all tools using libpod to manage containers.  This file is ignored when running in rootless mode.
+libpod.conf is the configuration file for all tools using libpod to manage containers.  When Podman runs in rootless mode, then the file `$HOME/.config/containers/libpod.conf` is used.
 
 **storage.conf** (`/etc/containers/storage.conf`)
 
 storage.conf is the storage configuration file for all tools using containers/storage
 
 The storage configuration file specifies all of the available container storage options for tools using shared container storage.
 
-**mounts.conf** (`/usr/share/containers/mounts.conf` and optionally `/etc/containers/mounts.conf`)
+When Podman runs in rootless mode, the file `$HOME/.config/containers/storage.conf` is also loaded.
+
+**mounts.conf** (`/usr/share/containers/mounts.conf` and optionally `/etc/containers/mounts.conf` and `$HOME/.config/containers/mounts.conf`)
```

Comment:
The "$HOME/..." directory is only used in a rootless mode right?  If so should we point that out here?  If so, I'd suggest sticking to the same approach as storage.conf, i.e. don't add it here but add a separate line.  Same question for line 154 about registries.conf.

Reply:
I've changed that both in the documentation and in the code so now the files under `$HOME` are used only when running in rootless mode.